test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
Please ensure that this file is properly ignored by git since I can still see it in the github repository. Remember, ignoring by git means that it should not be pushed and shown in your github repository. :wink: